app/assets/javascripts/curation_concerns/file_manager/sorting.es6

```diff
@@ -82,4 +87,40 @@ export default class SortManager {
       }
     ).toArray()
   }
+
+  get alpha_sort_button() {
+    return $("*[data-action='alpha-sort-action']")
+  }
+
+  initialize_alpha_sort_button() {
+    Blacklight.onLoad(() => {
```

Comment:
Actually you're right and this was causing a bug that I hadn't noticed where the button didn't work unless you reload the page.